src/identity/apis/org.ts

```diff
@@ -132,17 +138,18 @@ export const fetchDefaultAccountDefaultOrg = async (): Promise<
 }
 
 // fetch data regarding whether the user can create new orgs, and associated upgrade options.
-export const fetchOrgCreationAllowance = async () => {
-  const response = await getAllowancesOrgsCreate({})
+export const fetchOrgCreationAllowance =
+  async (): Promise<OrgCreationAllowances> => {
+    const response = await getAllowancesOrgsCreate({})
 
-  if (response.status !== 200) {
-    throw new GenericError(
-      'Failed to determine whether this user can create a new organization.'
-    )
-  }
+    if (response.status !== 200) {
```

Comment:
Do you know what's going on with the linter here? I don't see a new code block but it's indenting it more.

Reply:
line 142 `: Promise<OrgCreationAllowances>` is the only addition here. The indentations are most likely caused by prettier